productService/modules/getProductsById.ts

```diff
@@ -1,11 +1,20 @@
 import { APIGatewayProxyHandler } from 'aws-lambda';
-import products from '../mocks/products.json';
+import connectToDb from '../helpers/connectToDb';
 
 export const getProductsById: APIGatewayProxyHandler = async (event, _context) => {
   try {
+    const client = await connectToDb();
     const { productId } = event.pathParameters;
 
-    const searchedProduct = products.find(product => product.id === productId);
+    const query = {
+      text: 'select * from products where products.id=$1',
+      values: [productId]
+    }
+
+    const { rows: searchedProduct} = await client.query(query);
+    console.log('searchedProduct: ', searchedProduct);
+
+    console.log(`method getProductsById was called with next pathParameters: productId ${productId}`);
 
     if(!searchedProduct) {
       throw new Error('No such product');
```

Comment:
It's not a good practice to use `throw` as `goto` statement in `try ... catch` blocks. Here you know what should be send in response and you can simply return 
```
return {
      statusCode: 404,
      headers: {
        'Access-Control-Allow-Origin': '*',
        'Access-Control-Allow-Credentials': true,
      },
      body: JSON.stringify({ message: 'Product not found' }),
    };
```